staging/src/k8s.io/component-base/metrics/testutil/promlint.go

```diff
@@ -44,6 +44,7 @@ var exceptionMetrics = []string{
 	"aggregator_openapi_v2_regeneration_count",
 	"apiserver_admission_step_admission_duration_seconds_summary",
 	"apiserver_current_inflight_requests",
+	"apiserver_longrunning_requests",
```

Comment:
we shouldn't add this new metric to the exception list since it conforms with Prometheus best practices.

Reply:
I add it to here because the original one is here too. Can you tell me where I should put it?😳😳

Reply:
You don't have to put it anywhere. This is an exception list for metrics that don't follow the Prometheus best practices, so it only contains metrics for which the formating CI test is failing. Since your changes introduce a metric that follows the best practices, we don't have to add it here :smiley:

Reply:
oh, I get it. Thanks a lot!😊😊

Reply:
Done. Thanks for your help~